system/lib/libc/musl/arch/emscripten/bits/alltypes.h

```diff
@@ -85,9 +91,9 @@ typedef long suseconds_t;
 #if defined(__NEED_pthread_attr_t) && !defined(__DEFINED_pthread_attr_t)
 typedef struct {
     union {
-        int __i[10];
-        volatile int __vi[10];
-        unsigned __s[10];
+        int __i[9];
+        volatile int __vi[9];
+        unsigned __s[9];
     } __u;
```

Comment:
I'm not sure how to review this change. Why did 10 become 9?

Reply:
This comes from upstream.  They unified these definitions here:

https://github.com/emscripten-core/musl/commit/2d3083e746cf314c16048875e919305b1018bf8b

See the new common definition in `alltypes.h.in`.

It does looks like we have an existing bug where these are too small when building with MEMORY64, but that is a separate/unrealted issue.